packages/jupyter-ai/jupyter_ai/chat_handlers/default.py

```diff
@@ -1,10 +1,18 @@
-from typing import Dict, List, Type
+import time
+from typing import Dict, Type
+from uuid import uuid4
 
-from jupyter_ai.models import ChatMessage, ClearMessage, HumanChatMessage
+from jupyter_ai.models import (
+    AgentStreamChunkMessage,
+    AgentStreamMessage,
+    HumanChatMessage,
+)
 from jupyter_ai_magics.providers import BaseProvider
-from langchain.chains import ConversationChain, LLMChain
 from langchain.memory import ConversationBufferWindowMemory
```

Comment:
Is this still required?

Reply:
Great catch, thank you! I've removed all references of `ConversationBufferWindowMemory` in the latest revision.